packages/core/src/commands/network-generate.ts

```diff
@@ -536,12 +538,6 @@ export class Command extends Commands.Command {
             byteBuffer.writeByte(generatorByte);
         }
 
-        if (genesisBlock.blockSignature) {
```

Comment:
What was the issue with this block of code that it had to be removed?

Reply:
I think the code is unreachable - `blockSignature` is set to undefined on line 485 - then passed immediately to `this.getBlockId(block)`, which in turn immediately calls this function here.